ui/src/dashboards/apis/index.ts

```diff
@@ -1,6 +1,11 @@
-import AJAX from 'utils/ajax'
+import AJAX from 'src/utils/ajax'
 
-export function getDashboards() {
+import {AxiosResponse} from 'axios'
+import {DashboardsResponse} from 'src/types/apis/dashboards'
+
+export const getDashboards = (): Promise<
+  AxiosResponse<DashboardsResponse> | DashboardsResponse
```

Comment:
this resolves to a DashboardsReponse in some cases, and to a wrapped DashboardsResponse in others?

Reply:
yeah, it's due to the current complicated typing of our AJAX function. i spoke with @bthesorceror about this and it's typed this way because there's a split somewhere in the codebase in how AJAX is consumed. the AJAX functions (wherever its consumption bifurcates) should be cleaned up later, at which point this can be resolved as well.